extensions/ql-vscode/test/factories/authentication.ts

```diff
@@ -0,0 +1,42 @@
+import { retry } from "@octokit/plugin-retry";
```

Comment:
Same comment for this file as the other ones: should this be named `credentials.ts`?